docs/releases/3.1.txt

```diff
@@ -27,6 +27,27 @@ officially support the latest release of each series.
 What's new in Django 3.1
 ========================
 
+Automatic :class:`~django.apps.AppConfig` detection
+---------------------------------------------------
+
+Most pluggable applications define an :class:`~django.apps.AppConfig` subclass
+in an ``apps.py`` submodule. Many define a ``default_app_config`` variable
+pointing to this class in their ``__init__.py``.
+
+When the ``apps.py`` submodule exists and defines a single
+:class:`~django.apps.AppConfig`, Django 3.1 uses that configuration
+automatically, so you can remove ``default_app_config``. If needed, you can
+adjust this behavior with :attr:`~django.apps.AppConfig.default`.
+
+``default_app_config`` made it possible to declare only the application's path
+in :setting:`INSTALLED_APPS` (e.g. ``'django.contrib.admin'``) rather than the
+app config's path (e.g. ``'django.contrib.admin.apps.AdminConfig'``). It was
+introduced for backwards-compatibility with the former style, with the intent
+to switch the ecosystem to the latter, but the switch didn't happen.
+
+With the automatic discovery, most apps no longer need ``default_app_config``.
+As a consequence, it's deprecated.
```

Comment:
It's not "most apps" right? It's all apps. 

1. Automatic discovery of single AppConfig. 
2. Finer-grained control using `default`. 
3. Explicit class path in INSTALLED_APPS.